deeplearning/ml4pl/models/ggnn/ggnn_config.py

```diff
@@ -23,14 +23,24 @@
 
 class GGNNConfig(object):
   def __init__(
-    self, num_classes: int, has_graph_labels: bool, edge_type_count: int = 3
+    self,
+    num_classes: int,
+    has_graph_labels: bool,
+    edge_type_count: int = 3,
+    has_aux_input: bool = False,
   ):
+    # not implemented here, because not relevant:
+    # train_subset, random_seed,
+    ###############
+
     self.lr: float = FLAGS.learning_rate
-    self.clip_grad_norm: bool = FLAGS.clamp_gradient_norm  # use 6.0 as default! Set to 0.0 for no clipping.
+    self.clip_grad_norm: bool = FLAGS.clamp_gradient_norm  # use 6.0 as default when clipping! Set to 0.0 for no clipping.
 
     self.vocab_size: int = 8568
     self.inst2vec_embeddings = FLAGS.inst2vec_embeddings
     self.emb_size: int = 200
+
+    # TODO This should be turned off on devmap!
     self.use_selector_embeddings: bool = True
```

Comment:
This should be lifted into a flag to set to false on devmap. Or something more hacky.

Reply:
Done 491432b339cbeb46ab23cd885373ba0b2adb7a2c